gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@ metadata.format.version = "1.1"
 [versions]
 
 # Important dependencies
-data = "1.21.4-rv1"
+data = "1.21.4-dev"
```

Comment:
We need to double-check that, so far I know we don't rely on dev versions for production.

Reply:
Until a new release is published for the DataGen repo, I have to use the dev build.